.babelrc

```diff
@@ -7,7 +7,8 @@
 		} ]
 	],
 	"plugins": [
-		"transform-runtime"
+		"transform-runtime",
+		"transform-object-rest-spread"
```

Comment:
>You should search for `Object.assign` inside the project :P 

This seems like one of the more reasonable and stable non-spec proposals to consider. It's always felt like it's the final missing piece of what already exists with object/array destructuring and array spread.